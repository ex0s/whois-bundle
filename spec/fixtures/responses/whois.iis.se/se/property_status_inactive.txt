# Copyright (c) 1997- .SE (The Internet Infrastructure Foundation).
# All rights reserved.

# The information obtained through searches, or otherwise, is protected
# by the Swedish Copyright Act (1960:729) and international conventions.
# It is also subject to database protection according to the Swedish
# Copyright Act.

# Any use of this material to target advertising or
# similar activities is forbidden and will be prosecuted.
# If any of the information below is transferred to a third
# party, it must be done in its entirety. This server must
# not be used as a backend for a search engine.

# Result of search for registered domain names under
# the .SE top level domain.

# The data is in the UTF-8 character set and the result is
# printed with eight bits.

state:            system
domain:           example.se
holder:           -
admin-c:          -
tech-c:           -
billing-c:        -
created:          2000-11-01
modified:         -
expires:          2006-04-18
nserver:          -
dnssec:           unsigned delegation
status:           inactive
registrar:        CoreRegistry
reason:           SE Blocked, Example and test domains
