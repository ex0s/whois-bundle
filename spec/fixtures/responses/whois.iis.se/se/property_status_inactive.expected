#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %TIME{2000-11-01}

#updated_on
  %s == nil

#expires_on
  %s %TIME{2006-04-18}


#registrant_contacts
  %s %CLASS{array}
  %s == []

#admin_contacts
  %s %CLASS{array}
  %s == []

#technical_contacts
  %s %CLASS{array}
  %s == []

#registrar
  %s %CLASS{registrar}
  %s.name == "CoreRegistry"
